src/transaction.cc

```diff
@@ -1721,7 +1721,7 @@ std::string Transaction::toJSON(int parts) {
                 reinterpret_cast<const unsigned char*>("tags"),
                 strlen("tags"));
             yajl_gen_array_open(g);
-            for (auto b : a.m_tags) {
+            for (auto& b : a.m_tags) {
```

Comment:
```suggestion
            for (const auto& b : a.m_tags) {
```
Probably here you could use `const` too.